tests/testcases.py

```diff
@@ -1,109 +1,125 @@
+        "daylight change yesterday": datetime(today.year, today.month, today.day, 6) + relativedelta(days=-1),
+        "daylight change 01.01.2022": datetime(2022, 1, 1, 6),
+        "monday in two weeks": None,
+        "tuesday in two years": None,
+        "tomorrow 12 o'clock": datetime(today.year, today.month, today.day, 12) + relativedelta(days=1),
+        "tomorrow at 7pm": datetime(today.year, today.month, today.day, 19) + relativedelta(days=1),
+        "tomorrow at 17h": datetime(today.year, today.month, today.day, 17) + relativedelta(days=1),
+        "tomorrow at 17:13": datetime(today.year, today.month, today.day, 17, 13) + relativedelta(days=1),
+        "tomorrow at 17": datetime(today.year, today.month, today.day, 17) + relativedelta(days=1),
+        "tomorrow afternoon": datetime(today.year, today.month, today.day, 15) + relativedelta(days=1),
+        "at the next afternoon at tomorrow": datetime(today.year, today.month, today.day, 15) + relativedelta(days=1),
+        "the day before yesterday": datetime(today.year, today.month, today.day) + relativedelta(days=-2),
+        "the day after tomorrow": datetime(today.year, today.month, today.day) + relativedelta(days=2),
+        "a day before yesterday": datetime(today.year, today.month, today.day) + relativedelta(days=-2),
+        "a day after tomorrow": datetime(today.year, today.month, today.day) + relativedelta(days=2),
+        "day before yesterday": datetime(today.year, today.month, today.day) + relativedelta(days=-2),
+        "day after tomorrow": datetime(today.year, today.month, today.day) + relativedelta(days=2),
+        "two days after tomorrow": datetime(today.year, today.month, today.day) + relativedelta(days=3),
+        "two days before yesterday": datetime(today.year, today.month, today.day) + relativedelta(days=-3),
+    },
     # Datetime Delta
-    "at 9pm": datetime(today.year, today.month, today.day, 21),
-    "at 9:00pm": datetime(today.year, today.month, today.day, 21),
-    "at 10 in the evening": datetime(today.year, today.month, today.day, 22),
-    "5 in the morning": datetime(today.year, today.month, today.day, 5),
+    "datetime_delta": {
+        "at 9pm": datetime(today.year, today.month, today.day, 21),
+        "at 9:00pm": datetime(today.year, today.month, today.day, 21),
+        "at 10 in the evening": datetime(today.year, today.month, today.day, 22),
+        "5 in the morning": datetime(today.year, today.month, today.day, 5),
+    },
     # GitHub issue #78
-    "quarter past 5pm": datetime(today.year, today.month, today.day, 17, 15, 0),
-    "two quarters past 5h": datetime(today.year, today.month, today.day, 5, 30, 0),
-    "one quarter before 10pm": datetime(today.year, today.month, today.day, 21, 45, 0),
-    "ten quarters after 03:01:10am": datetime(today.year, today.month, today.day, 5, 31, 10),
-    "hour past christmas": datetime(today.year, 12, 25, 1),
-    "30 minutes past easter": None
+    "github_issue_#78": {
```

Comment:
This goes into absolute prepositions. Add it there at the end but still add the comment at the top of those